packages/main/src/Menu.ts

```diff
@@ -263,22 +266,72 @@ class Menu extends UI5Element {
 		return this.shadowRoot!.querySelector<ResponsivePopover>("[ui5-responsive-popover]")!;
 	}
 
-	get _menuItems() {
-		return this.items.filter((item): item is MenuItem => !item.isSeparator);
+	get _list() {
```

Comment:
Please start using the `@query` decorator for this kind of stuff, for example see Dynamic page:

```

	@query(".ui5-dynamic-page-scroll-container")
	scrollContainer?: HTMLElement;

	@query("[ui5-dynamic-page-header-actions]")
	headerActions?: DynamicPageHeaderActions;
	```

Reply:
Done.